tests/test_jetpack.py

```diff
@@ -328,20 +300,23 @@ def test_mismatched_db_hash():
 
     err = _do_test(MockXPI({"bootstrap.js": bootstrap,
                             "resources/bootstrap.js": bootstrap,
-                            "harness-options.json":
-                                json.dumps(harnessoptions)}))
-    print err.print_summary(verbose=True)
-    assert not err.failed()
+                            "harness-options.json": harnessoptions}))
+    assert err.failed()
+    assert (("testcases_frameworks", "unknown_file") in
+            (e["id"] for e in err.errors))
 
-    assert "jetpack_loaded_modules" in err.metadata
-    assert not err.metadata["jetpack_loaded_modules"]
-    assert "jetpack_identified_files" in err.metadata
+    assert "jetpack_loaded_modules" not in err.metadata
 
-    assert "jetpack_unknown_files" in err.metadata
-    unknown_files = err.metadata['jetpack_unknown_files']
-    nose.tools.eq_(len(unknown_files), 2)
-    nose.tools.ok_('bootstrap.js' in unknown_files)
-    nose.tools.ok_('resources/bootstrap.js' in unknown_files)
+    # There is currently no analog for this. Restore if we decide to
+    # create one.
+    # assert "jetpack_unknown_files" in err.metadata
```

Comment:
I don't think so. I was just leaving it there for reference in case it turns out that someone was using that property.